ixofoundation/ixo-webclient
# DEX: Implement breadcrumb navigation

[ticket: WEB-214](https://ixo.atlassian.net/browse/WEB-214)

## Work Done
- built breadcrumb on exchange as following pattern
 exchange / <asset name> / swap
- end extension changes as per select tag changes
- add redux for exchange

## Steps to Test
- check valid link url for each option
- check end extension changes on select onchange event triggers

## Screenshots
![image](https://user-images.githubusercontent.com/50171204/128247245-04d64e62-5e97-4d8c-8070-54e8030207b1.png)
